test/suites/smoke-test-common-solo/test-configuration-consistency.ts

```diff
@@ -42,27 +59,28 @@ describeSuite({
                 // we need to know whether a chain is assigned currently
                 const currentAuthorityAssignment = Object.fromEntries(
                     [
-                        ...(await api.query.tanssiAuthorityAssignment.collatorContainerChain(sessionIndex))
+                        ...(await apiAtBlock.query.tanssiAuthorityAssignment.collatorContainerChain(sessionIndex))
                             .unwrap()
                             .containerChains.entries(),
                     ].map(([key, value]) => [key.toString(), value.map((v) => v.toHuman())])
                 );
-                const currentAuthorities = await api.query.session.validators();
+                const currentAuthorities = await apiAtBlock.query.session.validators();
 
                 const currentCollatorNumber = Math.min(currentAuthorities.length, config.maxCollators.toNumber());
 
                 const maxParas = Math.trunc(
                     (currentCollatorNumber - config.minOrchestratorCollators.toNumber()) /
                         config.collatorsPerContainer.toNumber()
                 );
+                const maxPossibleAssignmentsBecauseOfCores = await getMaxPossibleAssignmentsBecauseOfCores(apiAtBlock);
```

Comment:
a few takes on this.

- pendingAssignment is done with the configuration present at the begining of this session
- liveAssingment is done with the configuration present at the beginning of the previous session.

therefore we should compare apples with apples in my opinion. I have 0 doubts this is already more robust than what we have but since we are changing this, better change it for good I guess